content/pages/constitution.md

```diff
@@ -0,0 +1,313 @@
+Author: Fatema Boxwala
+Title: Constitution
+hidden: true
+
+# Structure of Organization #
+
+The Women in Computer Science Undergraduate Subcomittee is a
+subcommittee of Women in Computer Science, a standing committee of the
```

Comment:
So Jo is currently the Director of Women in Computer Science (directorship title). As this director, she is the Chair of the Women in Computer Science Committee, a standing committee of SCS. I'd get clarification on that from her tbh.